device/ospfv3/interface/interface.yaml

```diff
@@ -0,0 +1,179 @@
+          x-field-uid: 4
+        capabilities:
+          description: >-
+            Container for OSPFv3 interface capabilities.
+          $ref: '#/components/schemas/Ospfv3Interface.Capabilities'
+          x-field-uid: 5
+        options:
+          description: >-
+            Container for OSPFv3 optional interface properties.
+          $ref: '#/components/schemas/Ospfv3Interface.Options'
+          x-field-uid: 6
+    
+    Ospfv3Interface.Area:
+      description: |-
+        Container for OSPFv3 Area ID identifies the routing area to which the host belongs.      
+      type: object
+      properties:
+        choice:
+          description: |-
+            The OSPFv3 Area ID identifies the routing area to which the host belongs. Area ID type can be following format.
+            - id: A 32-bit number identifying the area.
+            - ip: The Area ID in IPv4 address format.
+          type: string
+          default: id
+          x-field-uid: 1
+          x-enum:
+            id:
+              x-field-uid: 1
+            ip:
+              x-field-uid: 2
+        id:
+          description: >-
+            The Area ID.
+          type: integer
+          format: uint32
+          default: 0
+          x-field-uid: 2
+        ip:
+          description: >-
+            The Area ID in IPv4 address format.
+          type: string
+          format: ipv4
+          x-field-uid: 3
+
+    Ospfv3Interface.NetworkType:
+      description: >-
+        The OSPFv3 network link type options.
+          - Broadcast
+          - Point to Point
+      type: object
+      properties:
+        choice:
+          type: string
+          default: broadcast
+          x-field-uid: 1
+          x-enum:
+            broadcast:
+              x-field-uid: 1
+            point_to_point:
+              x-field-uid: 2
+        
+    Ospfv3Interface.Capabilities:
+      description: >-
+        Contains OSPFv3 interface capabilities.
+      type: object
+      properties:
+        hello_interval:
+          description: |-
+            The time interval, in seconds, between the Hello packets that
+            the router sends on the interface. Advertised in Hello packets
+            sent out this interface.
+          type: integer
+          format: uint32
+          default: 10
+          maximum: 65535
+          x-field-uid: 1
+        dead_interval:
+          description: |-
+            The time interval in seconds before the router's neighbors will declare
+            it down, when they stop hearing the router's Hello Packets.
+            Advertised in Hello packets sent out this interface.
+          type: integer
+          format: uint32
+          default: 40
+          maximum: 65535
+          x-field-uid: 2
+        link_metric:
+          description: |-
+            The cost of transmitting data on this link.
+          type: integer
+          format: uint32
+          default: 10
+          x-field-uid: 3
+        priority:
```

Comment:
Maybe priority can be clubbed with broadcast type of link since it is actually not a relevant field for p2p type.

Reply:
Keeping it open to clarify impact on UX.

Reply:
Updated.